extensions/3DTILES_bounding_volume_S2/README.md

```diff
@@ -0,0 +1,459 @@
+<!-- omit in toc -->
+# 3DTILES_bounding_volume_S2
+
+<!-- omit in toc -->
+## Contributors
+
+- Sam Suhag, Cesium
+- Sean Lilley, Cesium
+- Peter Gagliardi, Cesium
+
+<!-- omit in toc -->
+## Status
+
+Draft
+
+<!-- omit in toc -->
+## Dependencies
```

Comment:
@lilleyse should this list the S2 specification as a dependency? or is it enough to mention it in the overview?

Reply:
I think the S2 <-> implicit connection should be mentioned upfront here. It could look similar to how EXT_mesh_features does that:

https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_mesh_features#dependencies

Reply:
I think mentioning S2 in the overview is fine